java/src/main/java/HelloApi.java

```diff
@@ -0,0 +1,44 @@
+import io.sphere.sdk.client.BlockingSphereClient;
+import io.sphere.sdk.client.SphereClient;
+import io.sphere.sdk.client.SphereClientConfig;
+import io.sphere.sdk.client.SphereClientFactory;
+import io.sphere.sdk.products.ProductProjection;
+import io.sphere.sdk.products.search.ProductProjectionSearch;
+
+import java.io.IOException;
+import java.io.InputStream;
+import java.util.List;
+import java.util.Properties;
+import java.util.concurrent.TimeUnit;
+
+// This example uses the official Sphere Java client:
+// https://github.com/sphereio/sphere-jvm-sdk
+
+public class HelloApi {
+    public static void main(String[] args) throws Exception {
+        final SphereClientConfig config = loadCommercetoolsConfig();
+        final SphereClientFactory factory = SphereClientFactory.of();
+        final SphereClient underlyingClient = factory.createClient(config);//this client only works asynchronous
```

Comment:
I would personally write
```
final SphereClient underlyingClient = SphereClientFactory.of().createClient(config);
```
To save a line of code in order to make it faster to understand and read, which in this particular case is important.